graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlServiceBuilder.java

```diff
@@ -263,6 +265,14 @@ public GraphqlServiceBuilder useBlockingTaskExecutor(boolean useBlockingTaskExec
         return this;
     }
 
+    /**
+     * Sets whether the service supports WebSockets for subscriptions.
```

Comment:
What do you think of leaving a link so that people can refer to it?

```
Sets whether [[GraphQL over WebSocket Protocol](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md)] is supported.
```

Reply:
Done